.env.example

```diff
@@ -1,17 +0,0 @@
-# Database - change to fit your database
```

Comment:
Seems like you accidentally deleted this file.

Reply:
No, it's just renaming the file and removing the .example suffix

Reply:
I don't follow. Can you explain a bit more? How can they run my app by doing cp .env.example .env? This should replace .env content with .env.example that should most likely not work on everyone's device.
My understanding so far is that .env.example is like a template that is used as a guideline for starting development and after one puts his settings inside it then he can rename the file.

Reply:
How would someone run this mealsharing app when cloning this repo to their computer? The `.env` you have is only on your computer so we want to keep this `.env.example` so they can copy and rename it to `.env`. Does that make sense?

Reply:
Yes. Now I got it. Thank you so much. Thanks for your time and excellent explanations.